Zend/zend_simd.h

```diff
@@ -0,0 +1,83 @@
+/*
+   +----------------------------------------------------------------------+
+   | Zend Engine                                                          |
+   +----------------------------------------------------------------------+
+   | Copyright (c) Zend Technologies Ltd. (http://www.zend.com)           |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 2.00 of the Zend license,     |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | http://www.zend.com/license/2_00.txt.                                |
+   | If you did not receive a copy of the Zend license and are unable to  |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@zend.com so we can mail you a copy immediately.              |
+   +----------------------------------------------------------------------+
+   | Authors: Saki Takamachi <saki@php.net>                               |
+   +----------------------------------------------------------------------+
+*/
```

Comment:
I think it would be better to license it under BSD (there are already other files) and not give copyright to Zend Technologies.

Reply:
Although copyright probably doesn't matter so  I don't really mind. It's kind of the same like giving it to PHP Group. Once PHP Foundation will become legal entity, this should be much easier...

Reply:
Maybe add you as a copy right holder and just change it to BSD. It would be good because it will make easier to use this file elsewhere without pulling in Zend license...

Reply:
By elsewhere I mean outside php-src...

Reply:
I would like to potentially pull it to jso (project that I currently use for JsonSchema development). So it might be actually even better to license it under MIT.